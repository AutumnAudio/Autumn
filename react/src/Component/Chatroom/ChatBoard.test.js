import React from 'react'
import renderer from 'react-test-renderer'
import ChatBoard from './ChatBoard'

test('should be able to render chat board', () => {
    const chatMock = [{username: 'testUser',
                      message: 'some message'},
                      {username: 'testUser2',
                      message: 'another message'}]
    const component = renderer.create(
        <ChatBoard chat={chatMock} />,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
