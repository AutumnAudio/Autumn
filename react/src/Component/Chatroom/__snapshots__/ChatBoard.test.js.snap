// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be able to render chat board 1`] = `
<div
  className="ChatBoard-chatBoardDiv-1"
>
  <div>
    <div
      className="ChatMessage-chatMessageList-5"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root ChatMessage-personIcon-2"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
        />
      </svg>
      <div
        className="ChatMessage-nameDiv-3"
      >
        testUser
      </div>
      <div
        className="ChatMessage-textDiv-4"
      >
        some message
      </div>
    </div>
    <div
      className="ChatMessage-chatMessageList-5"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root ChatMessage-personIcon-2"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
        />
      </svg>
      <div
        className="ChatMessage-nameDiv-3"
      >
        testUser2
      </div>
      <div
        className="ChatMessage-textDiv-4"
      >
        another message
      </div>
    </div>
  </div>
</div>
`;
